packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -47,18 +47,24 @@ Future<Null> pubGet({
     return;
   }
 
-  if (!checkLastModified || _shouldRunPubGet(pubSpecYaml: pubSpecYaml, dotPackages: dotPackages)) {
+  if (!checkLastModified ||
+      _shouldRunPubGet(pubSpecYaml: pubSpecYaml, dotPackages: dotPackages)) {
     final String command = upgrade ? 'upgrade' : 'get';
     final Status status = logger.startProgress(
       'Running "flutter packages $command" in ${fs.path.basename(directory)}...',
       expectSlowOperation: true,
     );
-    final List<String> args = <String>['--verbosity=warning', command, '--no-precompile'];
+    final List<String> args = <String>[
+      '--verbosity=warning',
+      command,
+      '--no-precompile'
```

Comment:
Trailing comma (or just leave the line as it was)

Reply:
done